views/js/qtiCreator/editor/mathInput/mathInput.js

```diff
@@ -55,30 +55,99 @@ define([
+            perp2:      { label: '⊥',           latex: '\\perp',        fn: 'cmd',      desc: __('Perpendicular') },
+
+            // Basic geometry
+            triangle:   { label: '△',           latex: '\\triangle',    fn: 'cmd',      desc: __('Triangle') },
+            angle:      { label: '∠',           latex: '\\angle',       fn: 'cmd',      desc: __('Angle') },
+
+            // Lines and vectors
+            segment: {
+                label: '<math xmlns="http://www.w3.org/1998/Math/MathML"><mover accent="true"><mrow><mi>A</mi><mi>B</mi></mrow><mo>&#x27F7;</mo></mover></math>',
+                latex: '\\overleftrightarrow{AB}',
+                fn: 'write',
+                desc: __('Line Segment')
+            },
+            vector: {
+                label: '<math xmlns="http://www.w3.org/1998/Math/MathML"><mover accent="true"><mrow><mi>A</mi><mi>B</mi></mrow><mo>&#x27F6;</mo></mover></math>',
+                latex: '\\overrightarrow{AB}',
+                fn: 'write',
+                desc: __('Vector')
+            },
+            ray: {
+                label: '<math xmlns="http://www.w3.org/1998/Math/MathML"><mover accent="true"><mrow><mi>A</mi><mi>B</mi></mrow><mo style="margin-bottom: 0.2rem;">&#x21c0;</mo></mover></math>',
+                latex: '\\stackrel{\\rightharpoonup}{AB}',
+                fn: 'write',
+                desc: __('Ray')
+            },
+            line: {
+                label: '<math xmlns="http://www.w3.org/1998/Math/MathML"><mover accent="true"><mrow><mi>A</mi><mi>B</mi></mrow><mo style="margin-bottom: 0.2rem;">&#x2E3A;</mo></mover></math>',
+                latex: '\\overline{AB}',
+                fn: 'write',
+                desc: __('Line')
+            },
+            arc: {
+                label: '<math xmlns="http://www.w3.org/1998/Math/MathML"><mover accent="true"><mrow><mi>A</mi><mi>B</mi></mrow><mo stretchy="true" style="font-size: 150%">&#x2322;</mo></mover></math>',
+                latex: '\\overset{AB}',
+                fn: 'write',
+                desc: __('Arc')
+            },
+
+            // Greek letters
+            alpha:      { label: 'α',           latex: '\\alpha',       fn: 'cmd',      desc: __('Alpha') },
+            beta:       { label: 'β',           latex: '\\beta',        fn: 'cmd',      desc: __('Beta') },
+            theta:      { label: 'θ',           latex: '\\theta',       fn: 'cmd',      desc: __('Theta') },
+            delta:      { label: 'Δ',           latex: '\\Delta',       fn: 'cmd',      desc: __('Delta') }
         },
+
         allToolGroups = [
-            { id: 'functions',  tools: ['sqrt', 'frac', 'exp', 'log', 'ln'] },
-            { id: 'symbols',    tools: ['e', 'infinity', 'lbrack', 'rbrack'] },
-            { id: 'trigo',      tools: ['pi', 'sin', 'cos'] },
-            { id: 'comparison', tools: ['lte', 'gte'] },
-            { id: 'operands',   tools: ['times', 'divide', 'plusminus'] }
```

Comment:
We have to be sure that the old ids were not used by anything (for example, is there a qtiCreator config to turn groups on or off?)